index.html

```diff
@@ -0,0 +1,499 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>laslessVPN</title>
+    <link rel="stylesheet" href="./styles/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <img
+          src="./img/logo.svg"
+          width="149"
+          height="36"
+          alt="lasslessvpn-logo"
+        />
+        <ul class="header__menu">
+          <li class="header__item">
+            <a class="header__link" href="#">About</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Features</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Pricing</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Testimonials</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Help</a>
+          </li>
+        </ul>
+        <div class="container__button">
+          <button class="button button__sign-in" type="button">Sign in</button>
```

Comment:
What's up with these buttons? They suddenly stopped being a part of a header. Let's mark them up like this, instead of `container__button`:
```
- header__buttons
  - header__button header__button--sign-in button
  - header__button header__button--sign-up button
```

Reply:
+